README.md

```diff
@@ -137,6 +132,44 @@ Analytics.setGlobalProperties({
 
 Note that global properties can be overridden by parameters passed to `trackEvent` calls.
 
+### Initializing `infermedicaAnalytics` module
+
+You can decide if you want to use Firebase anonymously or as identified user. In order to use `infermedicaAnalytics` 
+module of this package you should call `initialize()` method and pass it an object which matches the following interface:
+
+```typescript
+interface IInitializeParams {
+  firebaseAuth?: Auth, // { Auth } from '@firebase/auth';
+  forceSignInAnonymously: boolean, // by default it's set to true
+}
+```
+
+In case you need to handle firebase-authentication on your side, do it and then initialize Analytics library by passing the firebaseAuth object as a property of passing object, e.g.:
+
+```javascript
+import { initializeApp } from 'firebase/app';
+import { getAuth } from 'firebase/auth';
+import { Analytics } from '@infermedica/modular-analytics';
+
+const firebaseConfig = { /* your firebase config */ }; 
+
+const firebaseApp = initializeApp(firebaseConfig);
+const firebaseAuth = getAuth(firebaseApp);
+Analytics.initialize({ firebaseAuth, forceSignInAnonymously: false });
+```
+
+If you decide to use firebase as anonymous user, you can call the `initialize()` method without passed params or with an object with single property, e.g.:
```

Comment:
`firebase as anonymous user` should be on top and it should be default way to use modular-analytics. Passing firebase is should be optional part of nodular-analytics and it should be after anonymous user